### PR TITLE
Fixed the header problem for the csv file

### LINKED ISSUE #7: Table headers can be added to log file multiple times
**Expected**: If the log file already exists, the table headers should not be appended to the file.

**Actual**: If the program is rerun after being run once, the table headers will be appending to the top of the file more than once.

### PR DESCRIPTION
**Pull request by 10KSA01**
I believe what the problem before was that it couldn't find file path which could be why it did not know whether the file existed or not.

Now I have added the log file path and a built-in function that checks if the file exists.

### CODE REVIEW

**Comment by guyus15** [comment]
Looks good. Can you undo the config.json change?

File: config.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   },
   "serial": {
     "com": {
-      "windows": "COM8",
+      "windows": "COM3",
```
Comment (guyus15) [inline config.json]: Can you you undo this change?

Comment (10KSA01) [reply config.json]: Sure
